arm/helpers/helpers.go

```diff
@@ -0,0 +1,47 @@
+package helpers
```

Comment:
I think we should give this package a more meaningful name, something that describes its use.  I'm bad at naming but will throw a few out here: template, deployment.

Reply:
Fixed